renderer/components/withIntl.js

```diff
@@ -14,12 +14,6 @@ const getLocale = () => {
   return navigatorLang.split('-')[0]
 }
 
-if (typeof window !== 'undefined' && window.ReactIntlLocaleData) {
-  Object.keys(window.ReactIntlLocaleData).forEach((lang) => {
-    addLocaleData(window.ReactIntlLocaleData[lang])
-  })
-}
-
```

Comment:
This is removed because `react-intl` no longer provides locale data or the method `addLocaleData` to add such locale data. It is expected that modern browsers already implement them and recommended polyfills are included. However, in our case, we render in a controlled environment (unlike explorer) and can afford to not have the polyfills as we remove this functionality.

Reply:
context: "Locale Data" refers to date formats, number formats that are specific to locales.